website_field_autocomplete_tag/README.rst

```diff
@@ -0,0 +1,73 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============================
+Website Field AutoComplete Tag
+==============================
+
+Extends Website Autocomplete field to allow for selecting of multiple
+results, aka tagging.
+
+Tags are represented as comma separated lists.
+
+Usage
+=====
+
+To use this module, you would follow the instructions provided in
+``website_field_autocomplete``.
+
+In order to activate the tagging feature, you should use the `data-is-tag`
+attribute. Following is an example:
+
+.. code:: xml
+
+    <label for="name">Name</label>
+    <input type="text"
+           name="name"
+           class="js_website_autocomplete"
+           data-query-field="name"
+           data-relate-recv="res_partner"
+           data-model="res.partner"
+           data-is-tag=""
```

Comment:
The attribute should have a value as made with invisible attribute in views (0 > False, other value, usually 1 > True)